frontend/public/components/edit-yaml.jsx

```diff
@@ -585,21 +594,64 @@ export const EditYAML = connect(stateToProps)(
       });
     }
 
+    toggleSidebar = () => {
+      this.setState(
+        (state) => {
+          return { showSidebar: !state.showSidebar };
+        }
+      );
+      window.dispatchEvent(new Event('sidebar_toggle'));
+    }
+
     render() {
       if (!this.props.create && !this.props.obj) {
         return <Loading />;
       }
 
-      const { connectDropTarget, isOver, canDrop } = this.props;
+      const { connectDropTarget, isOver, canDrop, create, yamlSamplesList } = this.props;
       const klass = classNames('co-file-dropzone-container', {
         'co-file-dropzone--drop-over': isOver,
       });
 
-      const { error, success, stale, yaml, height } = this.state;
-      const { create, obj, download = true, header } = this.props;
+      const { error, success, stale, yaml, height, showSidebar } = this.state;
+      const { create, obj, download = true, header, yamlSamplesList } = this.props;
       const readOnly = this.props.readOnly || this.state.notAllowed;
       const options = { readOnly, scrollBeyondLastLine: false };
       const model = this.getModel(obj);
+      const samples = getResourceSidebarSamples(model, yamlSamplesList);
+      const showSamples = !_.isEmpty(samples) && create;
+      const linkLabel = `View Schema ${showSamples ? 'and Samples' : ''}`;
+      const sidebarLink = <div className="co-p-has-sidebar__sidebar--hidden hidden-sm hidden-xs">
```

Comment:
`co-p-has-sidebar__sidebar--hidden` isn't a great name in light of the changes.  And a shared rule with the shortcuts link would be better.

Rather than hiding each link at mobile it would be better to hide `yaml-editor__icons`